.github/workflows/labeling.yml

```diff
@@ -1,9 +1,14 @@
 name: Labeling
 
 on:
-  schedule:
```

Comment:
Should we also keep the schedule? Just in case no one opens an issue or PR in a day?

Reply:
I don't think we should because with both `issues` and `pull_request_target` events enabled, the action should be triggered right after a new issue or PR has been opened.